docs/eventing/samples/cronjob-source/README.md

```diff
@@ -24,7 +24,7 @@ spec:
       revisionTemplate:
```

Comment:
Should the name above (on line 20) be updated from message-dumper to something like event-display? @Harwayne  Would that impact the rest of the code for the samples?

Reply:
Yes. `service.yaml` will need to change in the same way.